src/components/home.vue

```diff
@@ -1,21 +1,34 @@
 <template>
-    <div>
-
-    </div>
+  <div class="container">
+    <MovieAndTvShowList
+      :moviesEndpoint="'popular'"
+      :tvShowsEndpoint="'popular'"
+      :title="'Most Popular'"
+    ></MovieAndTvShowList>
+ <MovieAndTvShowList
+      :moviesEndpoint="'now_playing'"
+      :tvShowsEndpoint="'airing_today'"
+      :title="'Now Playing'"
+    ></MovieAndTvShowList> <MovieAndTvShowList
+      :moviesEndpoint="'upcoming'"
+      :tvShowsEndpoint="'on_the_air'"
+      :title="'Upcoming'"
+    ></MovieAndTvShowList> <MovieAndTvShowList
+      :moviesEndpoint="'top_rated'"
+      :tvShowsEndpoint="'top_rated'"
+      :title="'Top Rated'"
+    ></MovieAndTvShowList>
+  </div>
 </template>
 <script>
+import MovieAndTvShowList from "./MovieAndTvShowList.vue";
 export default {
-    data(){
-        return{
-
-        }
-    },
-    created(){
-    const API_KEY = "ffebf14b46dcd2b2bb0af17fdfffaa0c"
-    console.log(API_KEY)
+  components: {
+    MovieAndTvShowList: MovieAndTvShowList,
```

Comment:
You can pass only `MovieAndTvShowList` because when you pass variable with certain name in the object then it is going to assign it the same key as the variable name.
e.g.
`
components: {
  MovieAndTvShowList
}
`